<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>FilterCatalogBase</class>
  <widget class="QDialog" name="FilterCatalogBase" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>512</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Filter Catalog</string>
    </property>
    <layout class="QGridLayout" >
      <item row="1" column="0" >
        <widget class="QLabel" name="textLabel2" >
          <property name="text" >
            <string>Last Name</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>lastNameLineEdit_</cstring>
          </property>
        </widget>
      </item>
      <item row="1" column="1" >
        <widget class="QLineEdit" name="lastNameLineEdit" />
      </item>
      <item row="4" column="0" >
        <widget class="QLabel" name="textLabel5" >
          <property name="text" >
            <string>Study Config</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>studyConfigLineEdit_</cstring>
          </property>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="QLabel" name="textLabel4" >
          <property name="text" >
            <string>Medical Record Number</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>mrnLineEdit_</cstring>
          </property>
        </widget>
      </item>
      <item row="6" column="1" >
        <widget class="QLineEdit" name="studyLocationLineEdit" />
      </item>
      <item row="6" column="0" >
        <widget class="QLabel" name="textLabel7" >
          <property name="text" >
            <string>Study Location</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>lineEdit6</cstring>
          </property>
        </widget>
      </item>
      <item row="5" column="1" >
        <widget class="QLineEdit" name="studyNumberLineEdit" />
      </item>
      <item row="3" column="1" >
        <widget class="QLineEdit" name="mrnLineEdit" />
      </item>
      <item row="0" column="0" >
        <widget class="QLabel" name="textLabel1" >
          <property name="text" >
            <string>Study Type</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>studyTypeComboBox_</cstring>
          </property>
        </widget>
      </item>
      <item row="2" column="1" >
        <widget class="QLineEdit" name="firstNameLineEdit" />
      </item>
      <item row="5" column="0" >
        <widget class="QLabel" name="textLabel6" >
          <property name="text" >
            <string>Study Number</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>studyNumberLineEdit_</cstring>
          </property>
        </widget>
      </item>
      <item row="4" column="1" >
        <widget class="QLineEdit" name="studyConfigLineEdit" />
      </item>
      <item row="2" column="0" >
        <widget class="QLabel" name="textLabel3" >
          <property name="text" >
            <string>First Name</string>
          </property>
          <property name="buddy" stdset="0" >
            <cstring>firstNameLineEdit_</cstring>
          </property>
        </widget>
      </item>
      <item rowspan="1" row="7" column="0" colspan="2" >
        <widget class="Q3ButtonGroup" name="studyDateButtonGroup" >
          <property name="title" >
            <string>Study Date</string>
          </property>
          <widget class="QLabel" name="textLabel9" >
            <property name="geometry" >
              <rect>
                <x>346</x>
                <y>131</y>
                <width>93</width>
                <height>30</height>
              </rect>
            </property>
            <property name="text" >
              <string>and</string>
            </property>
            <property name="alignment" >
              <set>Qt::AlignCenter</set>
            </property>
          </widget>
          <widget class="QRadioButton" name="todaysStudiesRadioButton" >
            <property name="geometry" >
              <rect>
                <x>11</x>
                <y>63</y>
                <width>200</width>
                <height>28</height>
              </rect>
            </property>
            <property name="text" >
              <string>Today's Studies</string>
            </property>
            <property name="shortcut" >
              <string/>
            </property>
            <property name="buttonGroupId" >
              <number>1</number>
            </property>
          </widget>
          <widget class="QLabel" name="todayLabel" >
            <property name="geometry" >
              <rect>
                <x>217</x>
                <y>63</y>
                <width>222</width>
                <height>28</height>
              </rect>
            </property>
            <property name="text" >
              <string>today</string>
            </property>
          </widget>
          <widget class="QRadioButton" name="thisWeeksStudiesRadioButton" >
            <property name="geometry" >
              <rect>
                <x>11</x>
                <y>97</y>
                <width>200</width>
                <height>28</height>
              </rect>
            </property>
            <property name="text" >
              <string>This Wee&amp;k's Studies</string>
            </property>
            <property name="shortcut" >
              <string>Alt+K</string>
            </property>
            <property name="buttonGroupId" >
              <number>2</number>
            </property>
          </widget>
          <widget class="QLabel" name="thisWeekLabel" >
            <property name="geometry" >
              <rect>
                <x>217</x>
                <y>97</y>
                <width>340</width>
                <height>28</height>
              </rect>
            </property>
            <property name="text" >
              <string>this week</string>
            </property>
          </widget>
          <widget class="Q3DateEdit" name="beginDateEdit" >
            <property name="enabled" >
              <bool>false</bool>
            </property>
            <property name="geometry" >
              <rect>
                <x>217</x>
                <y>131</y>
                <width>123</width>
                <height>30</height>
              </rect>
            </property>
          </widget>
          <widget class="Q3DateEdit" name="endDateEdit" >
            <property name="enabled" >
              <bool>false</bool>
            </property>
            <property name="geometry" >
              <rect>
                <x>445</x>
                <y>131</y>
                <width>123</width>
                <height>30</height>
              </rect>
            </property>
          </widget>
          <widget class="QRadioButton" name="anyDateRadioButton" >
            <property name="geometry" >
              <rect>
                <x>11</x>
                <y>29</y>
                <width>200</width>
                <height>28</height>
              </rect>
            </property>
            <property name="text" >
              <string>An&amp;y Date</string>
            </property>
            <property name="shortcut" >
              <string>Alt+Y</string>
            </property>
            <property name="checked" >
              <bool>true</bool>
            </property>
            <property name="buttonGroupId" >
              <number>0</number>
            </property>
          </widget>
          <widget class="QRadioButton" name="datesBetweenRadioButton" >
            <property name="geometry" >
              <rect>
                <x>11</x>
                <y>132</y>
                <width>200</width>
                <height>28</height>
              </rect>
            </property>
            <property name="text" >
              <string>Dates &amp;Between</string>
            </property>
            <property name="shortcut" >
              <string>Alt+B</string>
            </property>
            <property name="buttonGroupId" >
              <number>3</number>
            </property>
          </widget>
        </widget>
      </item>
      <item rowspan="1" row="8" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QPushButton" name="clearFormButton" >
              <property name="text" >
                <string>Clear For&amp;m</string>
              </property>
              <property name="shortcut" >
                <string>Alt+M</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>210</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Maximum</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="OkButton" >
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="shortcut" >
                <string>Alt+O</string>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton" >
              <property name="text" >
                <string>Ca&amp;ncel</string>
              </property>
              <property name="shortcut" >
                <string>Alt+N</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item row="0" column="1" >
        <widget class="QComboBox" name="studyTypeComboBox" >
          <item>
            <property name="text" >
              <string>Any</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>Study</string>
            </property>
          </item>
          <item>
            <property name="text" >
              <string>Pre-registered</string>
            </property>
          </item>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>OkButton</tabstop>
    <tabstop>cancelButton</tabstop>
    <tabstop>studyTypeComboBox</tabstop>
    <tabstop>lastNameLineEdit</tabstop>
    <tabstop>firstNameLineEdit</tabstop>
    <tabstop>mrnLineEdit</tabstop>
    <tabstop>studyConfigLineEdit</tabstop>
    <tabstop>studyNumberLineEdit</tabstop>
    <tabstop>studyLocationLineEdit</tabstop>
    <tabstop>anyDateRadioButton</tabstop>
    <tabstop>beginDateEdit</tabstop>
    <tabstop>endDateEdit</tabstop>
    <tabstop>clearFormButton</tabstop>
  </tabstops>
</ui>
